x-pack/functionbeat/docs/deploying.asciidoc

```diff
@@ -0,0 +1,95 @@
+[id="deploy-to-cloud-provider"]
+[role="xpack"]
+=== Deploy {beatname_uc} to your cloud provider
+
+After configuring {beatname_uc} and defining cloud functions for the services
+you want to monitor, deploy the functions to your cloud provider. To do this,
+you can:
+
+* <<manager-deployment,Use the {beatname_uc} manager>> (good for getting
+  started), 
+* Or <<own-deployment,use your own deployment infrastructure>>.
+
+[[manager-deployment]]
+==== Use the {beatname_uc} manager
+
+Use the built-in manager to deploy, update, or delete {beatname_uc} functions
+when you don't have your own deployment infrastructure or process in place.  
+
+During deployment, the {beatname_uc} manager:
+
+* Exports an AWS CloudFormation template. You can inspect the template by
+running the <<export-command,`export`>> command.
+* Creates a zip package that includes the function code and +{beatname_lc}.yml+
+config file.
+* Uploads the package to the specified cloud provider.
+
+See <<{beatname_lc}-deploying>> in the getting started to learn how to deploy
+functions by using the {beatname_uc} manager.
+
+[[own-deployment]]
+==== Use your own deployment infrastructure
+
+If you don't want to use the {beatname_uc} manager, use your own deployment
+infrastructure. To do this, create a package, then deploy it to your cloud
+provider:
+
+. Set the following environment variables:
++
+`BEAT_STRICT_PERMS=false`:: This setting makes the function skip the ownership
+check on the configuration file.
+`ENABLED_FUNCTIONS=function-name-1,function-name-2`:: Specifies a comma-
+separated list of functions that are enabled in the configuration file. For
+example, to package functions called `my-kinesis` and `my-cloudwatch-logs`, run:
++
+*linux and mac*:
++
+[source, shell]
+----
+export BEAT_STRICT_PERMS=false
+export ENABLED_FUNCTIONS=my-kinesis,my-cloudwatch-logs
```

Comment:
I would recommend a lambda per function, they are just easier to manage that way.